src/App.js

```diff
@@ -4,19 +4,33 @@ import './App.scss';
 import todos from './api/todos';
 import users from './api/users';
 
+import { TodoList } from './components/TodoList';
+
+export const preparedTodos = todos.map((todo) => {
```

Comment:
```suggestion
const preparedTodos = todos.map((todo) => {
```
Можно не экспортировать переменную, ты её передаешь как `props` в этом же файле и нигде не принимаешь больше.

Reply:
agree, fix this